src/transformers/tokenization_utils_base.py

```diff
@@ -1569,6 +1569,13 @@ class PreTrainedTokenizerBase(SpecialTokensMixin, PushToHubMixin):
     def __init__(self, **kwargs):
         # inputs and kwargs for saving and re-loading (see ``from_pretrained`` and ``save_pretrained``)
         self.init_inputs = ()
+        if "add_special_tokens" in kwargs:
+            kwargs["_add_special_tokens"] = kwargs.pop("add_special_tokens")
```

Comment:
why are we saving this? It's never used btw no?

Reply:
I think you mentioned we might be able to use it.
https://github.com/huggingface/transformers/pull/31233#pullrequestreview-2125432908

Reply:
Yeah but if we don't then let's not save it!